src/components/ContextBar.js

```diff
@@ -65,58 +69,74 @@ export const ContextBar = ({
     [Utils.DEFAULT, () => colors.warning()])
 
   const getIconForTool = (toolLabel, status) => {
-    return div({ style: { display: 'flex', flexDirection: 'column' } }, [
-      div({ style: { display: 'flex', justifyContent: 'center', alignItems: 'center', marginTop: '1rem' } }, [
+    return h(Clickable, {
+      style: { display: 'flex', flexDirection: 'column', justifyContent: 'center', ...contextBarStyles.contextBarButton, borderBottom: '0px' },
+      hover: contextBarStyles.hover,
+      onClick: () => {
+        setSelectedToolIcon(toolLabel)
+        setCloudEnvOpen(true)
+      },
+      tooltipSide: 'left',
+      tooltip: `${toolLabel} Environment ( ${getComputeStatusForDisplay(status)} )`,
```

Comment:
Is the extra space around the parentheses needed?

```suggestion
      tooltip: `${toolLabel} Environment (${getComputeStatusForDisplay(status)})`,
```

Reply:
That's what the UX team requested